index.html

```diff
@@ -368,7 +371,7 @@ <h2 class="text-3xl text-gray-800 mb-6">Who's building this?</h2>
       >
         Remote Ruby
       </a>
-      podcast.
+      podcast?
```

Comment:
maybe we were just being super assertive you should checkout the podcast :D :trollface: